arup-group/a-carbon-tool
# fixed login bug (I think)

I think that this fixes the login redirect bug. The issue was that when people who weren't logged in were going to the site, they weren't being properly redirected to the login page. Not really sure why it was working in the first place tbh, and not entirely sure if I've fully fixed the issue. But this seems to work for me now.

Steps to test should be:

1. go into an incognito window to ensure that you're not logged in
2. go to localhost
3. see that you're redirected back to the login page
4. log in
5. refresh the page and check that you don't get redirected (as you're logged in)
